.github/workflows/publish.yml

```diff
@@ -0,0 +1,157 @@
+            --platform-version ${{ env.PLATFORM_VERSION }} \
+            --debug
+      - name: confirm packages
+        run: ls packaging
+      - name: Set pg-backup-api version
+        run: |
+          echo FULL_VERSION=$(cat pg_backup_api/version.txt) >> $GITHUB_ENV
+      - name: Push pg-backup-api DEB to CloudSmith Repo
+        id: push_pg_backup_api_deb
+        uses: cloudsmith-io/action@master
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: "push"
+          format: "deb"
+          owner: "enterprisedb"
+          repo: "${{env.CLOUDSMITH_REPO}}"
+          distro: "${{ matrix.platform_flavour }}"
+          release: "${{ matrix.platform_version }}"
+          version: "any-version"
+          republish: "true" # needed ONLY if version is not changing
+          file: >
+            packaging/pg-backup-api_${{env.FULL_VERSION}}-${{env.RELEASE_VERSION}}_amd64.deb
+
+  build-rpms:
+    name: build-rpms
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - python_version: 3.6
+            platform_flavour: el
+            platform_version: "8"
+          - python_version: 3.6
+            platform_flavour: el
+            platform_version: "7"
+          - python_version: 3.6
+            platform_flavour: sles
+            platform_version: "15"
+          - python_version: 3.6
+            platform_flavour: sles
+            platform_version: "12"
+    env:
+      RELEASE_VERSION: "1"
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install other deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ruby \
+                                  ruby-dev \
+                                  rubygems \
+                                  build-essential \
+                                  python3 \
+                                  virtualenv
+      - name: Install fpm
+        run: sudo gem install fpm
+      - name: Install python deps
+        run: sudo pip install virtualenv-tools3
+      - name: Install python
+        env:
+          PYTHON_VERSION: ${{ matrix.python_version }}
+        run: |
+          sudo apt-get install software-properties-common
+          sudo add-apt-repository ppa:deadsnakes/ppa
+          sudo apt-get install python${PYTHON_VERSION} python${PYTHON_VERSION}-dev
+      - name: Build rpm
+        working-directory: ./packaging
+        env:
+          PLATFORM_FLAVOUR: ${{ matrix.platform_flavour }}
+          PLATFORM_VERSION: ${{ matrix.platform_version }}
+          PYTHON_VERSION: ${{ matrix.python_version }}
+        run: |
+          ./pkg.sh rpm ${{env.RELEASE_VERSION}} \
+            --python ${{ env.PYTHON_VERSION }} \
+            --platform-flavour ${{ env.PLATFORM_FLAVOUR }} \
+            --platform-version ${{ env.PLATFORM_VERSION }} \
+            --debug
+      - name: confirm packages
```

Comment:
Do we still need this (and its deb match on 63)? I think I added them for my own debugging so I could just look at the log output for a quick check.

Reply:
We can remove this - I left it in because I also found it helpful when making changes to the workflow.